picker/picker.go

```diff
@@ -299,15 +303,15 @@ func (p *Picker) WaitForStateChange(ctx context.Context, sourceState grpc.Connec
 	// TODO(stevvooe): This is questionable, but we'll see how it works.
```

Comment:
This is still questionable... and we're probably getting rid of this after the move to grpc.LoadBalancer.